### PR TITLE
Update compelling example in README.md for v6.x

### PR DESCRIPTION
**Pull request by dlech**

### CODE REVIEW

**Comment by jlaanstra** [comment]
No one true C# style anymore? :)

File: README.md [hunk 1]
```diff
@@ -40,44 +40,44 @@ public class ColorChooserThatDoesntLikeGreen : ReactiveObject
...
 
-  ReactiveCommand OkButton { get; protected set; }
+  public ReactiveCommand<object> OkButton { get; protected set; }
 
   public ColorChooserThatDoesntLikeGreen()
   {
-    var finalColor = this.WhenAny(x => x.Red, x => x.Green, x => x.Blue, 
-        (r,g,b) => Color.FromRGB(r.Value, g.Value, b.Value));
+    var finalColor = this.WhenAny(x => x.Red, x => x.Green, x => x.Blue,
+      (r,g,b) => Color.FromArgb(r.Value, g.Value, b.Value));
```
Comment (jlaanstra) [inline README.md]: FromArgb also needs an alpha value right?

File: README.md [hunk 2]
```diff
@@ -40,44 +40,44 @@ public class ColorChooserThatDoesntLikeGreen : ReactiveObject
...
 
-    finalColor.ToProperty(this, x => x.Color, out _Color);
+    color= finalColor.ToProperty(this, x => x.Color);
 
     // When the finalColor has full green, the Ok button is disabled
-    OkButton = new ReactiveCommand(finalColor.Select(x => x.Green != 255));
+    OkButton = ReactiveCommand.Create(this.WhenAny(x => x.Green, x => x.Value != 255));
```
Comment (jlaanstra) [inline README.md]: Switching to Create is good, but can we keep the observable instead of doing a new WhenAny?

File: README.md (hunk 1 above)
Comment (dlech) [inline README.md]: There is an overload without the alpha parameter.

https://msdn.microsoft.com/en-us/library/cce5h557%28v=vs.110%29.aspx

File: README.md (hunk 2 above)
Comment (dlech) [inline README.md]: This did not compile, so I was unsure how to fix it correctly. How would one keep the observable?

Comment (dlech) [inline README.md]: Nevermind. Was missing `using System.Reactive.Linq;`

**Comment by dlech** [comment]
> No one true C# style anymore? :)

Alas, I have not been instructed in the ways of the One True C# style.


I've updated the pull request to fix the issues. I admit that there are several changes that involve the `Color` class that make this less readable (i.e. `.FromArgb` and `x.G`), but now you can copy and paste this example into a .cs file and it will compile.

**Comment by jlaanstra** [comment]
I think it looks good now. I leave it to @ghuntley to merge as he assigned it to himself.

**Pull request merged**